fastlane/Fastfile

```diff
@@ -10,16 +10,27 @@
 #     https://docs.fastlane.tools/plugins/available-plugins
 #
 
-files_with_version_number = [
-  './.version',
-  './PurchasesHybridCommon.podspec',
-  './android/build.gradle',
-  './android/gradle.properties'
-]
-files_with_version_number_without_prerelease_modifiers = [
-  './ios/PurchasesHybridCommon/Info.plist',
-  './ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Info.plist'
-]
+files_with_version_number = {
+    './.version' => ['{x}'],
+    './PurchasesHybridCommon.podspec' => ['s.version          = "{x}"'],
+    './android/build.gradle' => ['versionName "{x}"'],
+    './android/gradle.properties' => ['VERSION_NAME={x}']
+}
+PLIST_VERSION_PATTERNS = ['<string>{x}</string>']
```

Comment:
This seems like a very generic pattern. Is there any chance of conflict and substituting more than what we need in the plist files? This might work if we are substituting using the current version number but if we want to switch to use a generic semver version, this being so generic might cause problems

Reply:
ohh I was thinking the same and just rechecked and it looks like we can use the line above to make sure we are updating the right version

```
	<key>CFBundleShortVersionString</key>
	<string>4.18.0</string>
```